recipes/reduct-cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,18 @@
+#include "reduct/client.h"
+
+#include <iostream>
+
+using reduct::IBucket;
+using reduct::IClient;
+
+int main() {
+    auto client = IClient::Build("http://127.0.0.1:8383");
```

Comment:
Can we avoid hitting the network layer, even if it is just locally? Thanks!

Reply:
@RubenRBS 
Thank you for your comment.
Fixed.